packages/cli/tests/subjects/progressive-hydration-preact8/index.js

```diff
@@ -0,0 +1,19 @@
+import Home from './routes/home';
+
+if (typeof window !== 'undefined') {
+	window.ROOT_MUTATION_COUNT = 0;
+	const root = document.querySelector('#app');
+	const old = root.removeChild;
+	root.removeChild = child => {
+		window.ROOT_MUTATION_COUNT++;
+		old(child);
```

Comment:
this would throw (thankfully it's not being called! haha)

```suggestion
		old.call(this, child);
```

Reply:
done